iree/modules/strings/BUILD

```diff
@@ -31,6 +31,13 @@ cc_library(
     ],
 )
 
+iree_cmake_extra_content(
+    content = """
+if(${IREE_BUILD_COMPILER} AND ${IREE_BUILD_TESTS})
```

Comment:
Drop the indent. It makes the indenting in the cmake file weird. I suggested `textwrap.dedent` to Scott when this was introduced for this reason

Reply:
Like this?

<img width="607" alt="Untitled" src="https://user-images.githubusercontent.com/59985653/102957452-4d21b980-448f-11eb-84d0-9bbeb6a560ae.png">

Reply:
Uh not sure how that will work with the newlines. If it works, then great :-) I was thinking like https://github.com/google/iree/blob/4ca12072652be6785075e8e6e48052347f9de784/iree/hal/vulkan/BUILD#L26-L30 The main thing is just not having the if be indented weird. in the CMake file.

Reply:
Wait I just read this again and now I see that GitHub just makes everything look indented 2 spaces... I have it backwards

The thing that is weird is that the stuff inside the if block in the cmake file isn't indented, which makes it hard to track. I don't see an easy way to fix that, so just ignore me.

Reply:
One way around this is to make this a guard statement instead and put all the compiler stuff at the bottom. Then this would be

```suggestion
if(NOT ${IREE_BUILD_COMPILER})
  return()
endif()
```

Reply:
I switched to this for all files except iree/vm/BUILD, where it was a bit more complicated